src/sentry/relocation/tasks/process.py

```diff
@@ -488,6 +507,12 @@ def cross_region_export_timeout_check(
     retry_backoff=RETRY_BACKOFF,
     retry_backoff_jitter=True,
     soft_time_limit=FAST_TIME_LIMIT,
+    taskworker_config=TaskworkerConfig(
+        namespace=relocation_tasks,
```

Comment:
Should we have a processing_deadline of FAST_TIME_LIMIT to align with the celery configuration?